ResNet_constructor.ipynb

```diff
@@ -0,0 +1,1393 @@
+{
```

Comment:
<p>Why you set some parameters at the top (like learning rate), but others hardcode here (like decay) ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bestetc/May/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
<p>I could fully parametrize optimizer and sheduler and there are no diff with full hardcode. </p><p>I try to parametrize most sense params to control it from the hyperparameter's cell and other params explicit in the functional body.</p>